### ISSUE TITLE
Shapes aren't displayed despite shape attr existing

### ISSUE THREAD

**Issue by vprajapati-tt** [tenstorrent/model-explorer#13]
- Unsure of why exactly this is happening, the `shape` special attr is in the format of a Python list str output (ex: [1, 2]) and it isn't being picked up or displayed on "show-tensor-shape" option being clicked.

**Comment by madcampos** [comment]
@vprajapati-tt could you provide an example file and point to how to reproduce the issue please?

**Comment by madcampos** [comment]
I think this is related to https://github.com/tenstorrent/tt-mlir/issues/1212

**Comment by vprajapati-tt** [comment]
yeah they are related. I tried messing around with the backend. At this point it seems to be some sort of frontend issue, I don't know why we're not triggering the mechanism which loads those `shape`values. It would be nice if we could investigate this.
